test/kwtool/crawlers_test.exs

```diff
@@ -23,4 +23,44 @@ defmodule Kwtool.CrawlersTest do
       assert keyword3.status == "added"
     end
   end
+
+  describe "get_all_user_keywords/2" do
+    test "returns a list of keywords when given a user" do
+      created_user = insert(:user)
+      insert(:keyword, user: created_user)
```

Comment:
Maybe insert another keyword that does not belong to the user. to make sure it is not included in the result list.

Reply:
yes, i added it below! Thanks 👍

Reply:
I feel like we can make it in the same test and probably use a name like `returns a list of keywords that belongs to the given user`